### PR TITLE
Update policies for OTC changes

### PR DESCRIPTION
**Pull request by mattcaswell**
Update other policies as necessary to reflect the bylaws changes that
introduced the OTC concept.

### CODE REVIEW

**Review by t-j-h — APPROVED** [review]

**Review by paulidale — APPROVED** [review]

File: policies/committers.html
Comment (mspncp) [inline policies/committers.html]: I don't think think that this addition is useful, it just distracts from the clear and concise statement of the sentence. (IOW I would revert the change.)

A useful description of _how_ the code is contributed would require a lot more words, but that is outside the scope of this policy document. And just adding a laconic "(GitHub pull requests)" does not really explain anything.

**Review by mspncp — COMMENTED** [review]
LGTM, except for one change.

File: policies/sidebar.shtml
Comment (mspncp) [inline policies/sidebar.shtml:25]: Maybe rename the file to `openssl-bylaws.html` ?

File: policies/committers.html
Comment (mattcaswell) [reply policies/committers.html]: I reverted this change since it is entirely tangential to the main point of the PR anyway.

File: policies/sidebar.shtml
Comment (mattcaswell) [reply policies/sidebar.shtml:25]: While I have no objection in principle to this, it turns out to be non-trivial since this page is referenced by quite a few other pages - so I'd rather keep it as-is for now.

**Comment by mattcaswell** [comment]
I pushed a fixup addressing the comment above.

**Review by mspncp — APPROVED** [review]

File: policies/sidebar.shtml
Comment (richsalz) [reply policies/sidebar.shtml:25]: You can always use the new name, and have your webmaster set up a redirect in .httaccess for the old name.

**Review by levitte — APPROVED** [review]

**Comment by levitte** [comment]
Hmm, why is @mspncp's approval not visible? Github, what are you up to?

**Comment by mspncp** [comment]
> Hmm, why is @mspncp's approval not visible? Github, what are you up to?

It's there, but the check mark is greyed, because I'm only a committer and this is the web repository ;-)

**Comment by mspncp** [comment]
(It's only listed in the top right of the page, not beneath "Changes approved")

**Comment by mspncp** [comment]
Worx as designed ;-)

**Comment by levitte** [comment]
> and this is the web repository

Doh! Of course

**Comment by kroeckx** [comment]
Should we vote on this?

**Review by mspncp — APPROVED** [review]
Reapproved, this time for real. Hope this does not create a logical circle :-P

**Comment by mspncp** [comment]
> Should we vote on this?

Since it is a consequence of the bylaw changes which were voted on by the OMC, it might be sufficient if you just add your own approval, then this pr will have four OMC approvals? Just and idea.

**Review by paulidale — APPROVED** [review]
I don't think a vote is required.  These aren't by laws changes.

**Comment by mattcaswell** [comment]
> I don't think a vote is required. These aren't by laws changes.

IMO, this *does* require a vote

**Comment by mattcaswell** [comment]
I started a vote on this:

> Update various project policies to the text shown here:
>        https://github.com/openssl/web/pull/146 (as of commit f432359)

**Comment by romen** [comment]
> I started a vote on this:
> 
> > Update various project policies to the text shown here:
> > #146 (as of commit [f432359](https://github.com/openssl/web/commit/f43235924ab058f9559382efbeb4454508b5bc26))

For future votes, it is possible to link to the "Changes" view up to a given commit in a given PR: https://github.com/openssl/web/pull/146/files/f43235924ab058f9559382efbeb4454508b5bc26

**Comment by t-j-h** [comment]
A vote is require to change policies. Always has been.

All official decisions are on the basis of a vote.

Tim.

**Comment by romen** [comment]
While the vote is under way, we have a few issues that could use some OMC ~~love~~ reviews :)

If anyone can spare some time:
- [ ] https://github.com/openssl/openssl/pull/9239
- [ ] https://github.com/openssl/openssl/pull/10693
- [ ] https://github.com/openssl/openssl/pull/10710

**Comment by t-j-h** [comment]
Note the vote to approve this change has passed. It is just awaiting merging.
I closed the vote as it had reached the required number of votes and it is blocking OTC members from contributing to reviews ...

**Comment by mattcaswell** [comment]
Pushed!!

**Pull request closed without merging**
